Packs/Axonius/Integrations/Axonius/Axonius.yml

```
@@ -29,7 +29,7 @@ script:
     - description: The name of the devices saved query within Axonius. See https://docs.axonius.com/docs/saved-queries-devices
       name: saved_query_name
       required: true
-    - description: The maximum number of results to return.
+    - description: The maximum number of results to return (Default - 50).
       name: max_results
```

```suggestion
    - description: The maximum number of results to return.
      name: max_results
      defaultValue: "50"
```
You can add the field `defaultValue` and in XSOAR's UI it will automatically add `Default - 50` to the description.
Of course this also refers to the other commands below.

---

@sahar-hagbi1 - You need to rerun the `demisto-sdk generate-docs -i Packs/Axonius/Integrations/Axonius/Axonius.yml` for the default value to be added to the README.